html/ui/js/brs.modals.transaction.js

```diff
@@ -903,8 +903,11 @@ var BRS = (function(BRS, $, undefined) {
                     else {
 			message = String(transaction.attachment.message);
 		    }
-
-		    $("#transaction_info_output_bottom").append("<div style='padding-left:5px;'><label><i class='fas fa-unlock'></i> " + $.t("public_message") + "</label><div>" + String(message).escapeHTML().nl2br() + "</div></div>");
+            if (transaction.type === 22 && transaction.subtype === 1) {
```

Comment:
Does this actually fix anything? This should only matter if `transaction.type` is sometimes `'22'` (string) and sometimes `22` (number), and we care about that distinction.

Reply:
This will, upon transaction type 22/1 (AT transaction), ignore the transaction's `isText` being set to false, and display both as string as hex, as ATs have no way of setting the isText flag

Reply:
All type checks are numeric...I assume that number is guaranteed.... but hell, this entire file is a mess

Reply:
This seems different from when I first reviewed it. Originally it seemed to change a double equal to a triple equal. It's possible I was misreading it originally.